config/locales/interface/output_element_series/nl_labels.yml

```diff
@@ -143,10 +150,10 @@ nl:
         other_final_demand_steam_hot_water_input_curve: 'Overig'
         energy_direct_air_capture_in_collective_heat_network_mekko: "Direct Air Capture (CCUS)"
         energy_chp_combined_cycle_network_gas_in_heat_network_mekko: "Gas WKK - grootschalig"
-        energy_chp_supercritical_waste_mix_in_heat_network_mekko: "Afval WKK - grootschalig"
-        energy_chp_ultra_supercritical_coal_in_heat_network_mekko: "Kolen WKK - grootschalig"
-        energy_chp_ultra_supercritical_cofiring_coal_in_heat_network_mekko: "Bijstook kolen WKK - grootschalig"
-        energy_chp_ultra_supercritical_lignite_in_heat_network_mekko: "Bruinkool WKK - grootschalig"
+        energy_chp_supercritical_waste_mix_in_mekko_of_collective_heat: "Afval WKK - grootschalig"
+        energy_chp_ultra_supercritical_coal_in_mekko_of_collective_heat: "Kolen WKK"
+        energy_chp_ultra_supercritical_cofiring_coal_in_mekko_of_collective_heat: "Kolen WKK met bijstook"
+        energy_chp_ultra_supercritical_lignite_in_mekko_of_collective_heat: "Bruinkool WKK"
```

Comment:
These labels do not match entirely with the sliders in the front-end. I suggest aligning them. For example, instead of "Gas WKK - grootschalig" I would use "Gas STEG WKK".